content/_data/events/scale15x.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "Scale 15x"
+date: "2017-03-02 thru 03-05T09:00:00"
```

Comment:
FWIW events like FOSDEM are always specified with their first date, some time in the morning:

https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/events/fosdem-2016.adoc
https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/events/Fosdem2017.adoc